utils/globals.py

```diff
@@ -30,7 +33,63 @@ def set_lab(lab):
 def get_lab():
     if _LAB is None:
         modlog.error('get_lab called before set_lab')
-        #modlog.error('An unexpected error occurd')re
         sys.exit(1)
     return _LAB
 
+
+def set_sampler(sampler_name, version):
+    ''' sets the name and version of the sampler as the sampler UID
+
+    :param sampler_name: name of the method used for samplng
+    :param version: version of the method used for sampling reported
+                    by the sampling function
+    :returns: **no returns, call func get_sampler_uid
+    '''
+    global _SAMPLER, _SAMPLER_has_been_set, _SAMPLER_UID
+
+    _SAMPLER = sampler_name
+    _SAMPLERVERSION = version
+    _SAMPLER_UID = 'escalate_' + str(sampler_name) + '_' + str(version)
+
+    _SAMPLER_has_been_set = True
+
+
+def get_sampler_uid():
+    '''returns the sampler used for automated sampling
+    '''
+    global _SAMPLER_UID
+    if _SAMPLER is None:
+        modlog.error('get_sampler_uid called before set_sampler')
+        sys.exit(1)
+    return _SAMPLER_UID
+
+
+def get_sampler_author():
+    import devconfig
+    sampler_author = 'escalate_' + str(devconfig.RoboVersion)
+    return(sampler_author)
```

Comment:
This function does not reflect the two samplers. They have gotten combined into the name `escalate_RoboVersion`. You could have `escalate_devconfig.sampler_devconfig.RoboVersion` to keep that information,

Furthermore, only need to `import` less. 

`from devconfig import RoboVersion`

Reply:
This doesn't do what you think.  This writes out the **participantname** column as escalate_version.  The model name (i.e. the sampling method) still has a unique set of information with it.